### DISCUSSION TITLE
HAFAS Issue: Keine Abfahrten / No Departures

### DISCUSSION THREAD

**Discussion by MrKrisKrisu** [Traewelling/traewelling#3024] [Announcements]
_[english version below]_
Zur Zeit kommt es immer wieder vor, dass Träwelling von HAFAS den Fehler 502 als Antwort auf die Fahrtenanfrage erhält, was bei uns zu der Fehlermeldung „Für diese Uhrzeit/diesen Filter gibt es derzeit leider keine Abfahrten.” führt.

Dieser Fehler tritt immer mal wieder für ein paar Minuten auf. Versuche also bitte gegebenenfalls ein paar Minuten später noch einmal, die Abfahrten zu laden.

Warum das passiert, darüber können wir nur spekulieren - wir hoffen einfach, dass sich das Problem bald „von selbst” löst.

-----------------

At the moment it happens again and again that Träwelling receives the error 502 from HAFAS in response to the journey request, which leads to the error message “Unfortunately there are currently no departures for this time/this filter”.

This error occurs from time to time for a few minutes. So please try again a few minutes later to load the departures.

We can only speculate as to why this happens - we simply hope that the problem will soon resolve itself.

**Comment by clyaIO** [comment]
Well, maybe we need another data source. Apparently HAFAS will be deactivated?!

This is from an old version of the DB Navigator:

![image](https://github.com/user-attachments/assets/1821d775-b124-4922-a460-ab97287a34e9)
